env/includes/settings/tk-alias.yml

```diff
@@ -70,4 +73,7 @@ settings.tk-alias.project:
     tk-multi-workfiles2: "@settings.tk-multi-workfiles2.alias"
   menu_favourites:
   - {app_instance: tk-multi-workfiles2, name: File Open...}
+  new_file_template:
```

Comment:
I think you don't need to define the template for the project environment, as the "new_file" operation will only happen in the asset.step context (for me, the context is changed before calling the hook so this one should be executed in the right context but I may be wrong)

Reply:
We need to define the template for the project environment, an example where the project env is used is on opening Alias for the first time and on selecting a task in workfiles2 and then clicking "New File".